ssz/sedes/serializable.py

```diff
@@ -27,11 +35,13 @@
 TSerializable = TypeVar("TSerializable", bound="BaseSerializable")
 
 
-class MetaBase:
-    fields = None
-    field_names = None
-    field_attrs = None
-    container_sedes = None
+class Meta(NamedTuple):
+
```

Comment:
minor but i feel like we usually have no whitespace b/t class decl and class attrs

Reply:
:+1: I don't like how it looks, but I do like conventions, so I removed the new lines for all classes I could find.